aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -291,17 +295,33 @@ module aptos_framework::account {
 
     #[view]
     public fun exists_at(addr: address): bool {
```

Comment:
it's a bit odd to change this function to always return true, instead of having a new function - and moving callers to it.

Reply:
usually this function is called by a contract and if it returns false, they will abort.
We can have a new function but we cannot modify the existing contracts logic. How can we move callers to another functino w/o breaking existing ones?

Reply:
Most contracts actually do not abort when this returns false but create the account (technically the resource) instead. Is modifying this function important anymore if that's the case?